selfdrive/car/hyundai/interface.py

```diff
@@ -177,7 +177,7 @@ def _get_params(ret, candidate, fingerprint, car_fw, experimental_long, docs):
       ret.steerRatio = 13.75
       ret.tireStiffnessFactor = 0.5
     elif candidate in (CAR.KIA_K5_2021, CAR.KIA_K5_HEV_2020):
-      ret.mass = 3228. * CV.LB_TO_KG
+      ret.mass = 3228. * CV.LB_TO_KG + STD_CARGO_KG
```

Comment:
```suggestion
      ret.mass = 3381. * CV.LB_TO_KG + STD_CARGO_KG
```
increased a bit for the hybrid (avg)